modules/java_api/cpp/ov_compiled_model.cpp

```diff
@@ -0,0 +1,27 @@
+#include <jni.h> // JNI header provided by JDK
+#include "openvino/openvino.hpp"
+
+#include "openvino_java.hpp"
+#include "jni_common.hpp"
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_CompiledModel_CreateInferRequest(JNIEnv *env, jobject obj, jlong addr){
```

Comment:
`create_infer_request`

Reply:
This is against [the rules for naming native methods](https://docs.oracle.com/en/java/javase/17/docs/specs/jni/design.html) :
 We use the underscore (“_”) character as the substitute for the slash (“/”) in fully qualified class names.

Reply:
Got it, thanks! Mark related comments as resolved

Reply:
I can rename native methods, but to use the underscore I need to use `_1`, for example: `read_1model`.

Reply:
No, let's keep current approach